lib/ex_machina/ecto.ex

```diff
@@ -133,4 +134,10 @@ defmodule ExMachina.Ecto do
       {name, _type} -> Map.delete(map, name)
     end
   end
+
+  defp drop_nil_values(map) do
+    map
+    |> Enum.filter(fn({_, v}) -> v != nil end)
```

Comment:
I like this 👍

I feel that using `Enum.reject` would make the intention a bit clearer. What do you think?